SillyBusinessInc/SillyBusinessGame
# Feat/#50 room generation

## Purpose of this PR:
Added room layout generation. Still needs work but this is a working version that can be used for other tickets

## How to test:
Make sure that there is an object in your scene with the GameManagerReference script attached to it. Then you can simply use the Table class to generate a table.
### links: 
[Ticket](https://github.com/orgs/SillyBusinessInc/projects/1/views/1?pane=issue&itemId=86883352&issue=SillyBusinessInc%7CSillyBusinessGame%7C50)  
_I will add documentation when the system is finished_
